jumanji/training/setup_train.py

```diff
@@ -72,7 +73,7 @@ def setup_logger(cfg: DictConfig) -> Logger:
     elif cfg.logger.type == "neptune":
         logger = NeptuneLogger(
             name=cfg.logger.name,
-            project="InstaDeep/jumanji",
+            project="InstaDeep/quarks-search",
```

Comment:
Please revert back to jumanji
```suggestion
            project="InstaDeep/jumanji",
```